proto/hummock.proto

```diff
@@ -514,6 +509,14 @@ message SetCompactorRuntimeConfigRequest {
 
 message SetCompactorRuntimeConfigResponse {}
 
+message JavaPinVersionRequest {
```

Comment:
I think we can remove the term `java` since the behavior of `pin version` is not bounded to be used in only java.